docs/questions/sharing/alerts.md

```diff
@@ -78,10 +78,15 @@ Admins get special privileges with alerts.
 - Everyone can add any Metabase user account, email address, or even a Slack channel as a recipient of an alert that they created (but not alerts created by others).
 - Everyone can view and unsubscribe from all alerts they receive by clicking on the **gear** icon in the upper right and navigating to **Account settings** > **Notifications**.
 
+## Avoid changing the name of the alerted channel in Slack
+
+Once you set up an alert to a Slack channel, avoid changing the name of that channel in Slack. If you rename the channel in Slack, but you want Metabase to continue to send alerts to that renamed channel, you'll need to update the alert in Metabase to point to the new channel name. 
+
 ## Alert expiration
 
 Some circumstances will automatically delete alerts:
 
+- Renaming the alerted channel in Slack.
```

Comment:
Does the alert get deleted because of channel renaming?

Or does the alert appear “active” — and it just doesn’t send anything?

Reply:
Ideally, if Metabase fails to send to a Slack channel, Metabase should email whomever set up the alert/subscription.